dbt/models/postgres/staging/stg_programs.sql

```diff
@@ -0,0 +1,33 @@
+{{ config(
+    materialized='view',
+    description='Staging model for translations_translation and translations_translation_translation tables'
+) }}
+
+WITH translations_t AS (
+    SELECT
+        tt.id AS translation_id,
+        tt.label
+    FROM {{ source('django_apps', 'translations_translation') }} tt
+    WHERE tt.label ILIKE 'program.%-name'
+       OR tt.label ILIKE 'program.%-apply_button_link'
+       OR tt.label ILIKE 'program.%-value_type'
+),
+translations_tt AS (
+    SELECT
+        ttt.id AS translation_translation_id,
+        ttt.master_id,
+        ttt.language_code,
+        ttt.text
+    FROM {{ source('django_apps', 'translations_translation_translation') }} ttt
+    INNER JOIN translations_t tt ON ttt.master_id = tt.translation_id
+    WHERE ttt.language_code = 'en-us'
+)
+
+SELECT
+    ttt.translation_translation_id,
+    ttt.master_id,
+    tt.label,
+    ttt.language_code,
+    ttt.text
+FROM translations_t tt
+INNER JOIN translations_tt ttt ON tt.translation_id = ttt.master_id
```

Comment:
This matches the materialized view exactly! But after thinking through it some more, I think it would be really valuable to add white_label_id, so that we can add rls and be able to filter these program values by white label.

I think you can do that by adding a join in stg_programs to the programs_program table id which has white_label_id. Then you can pass white_label_id up to the mart and add the rls post_hook.

```
INNER JOIN {{ source('django_apps', 'programs_program') }} pp ON tt.translation_id = pp.value_type_id
```